### PR TITLE
COVID-19 in India - An Analysis

### PR DESCRIPTION
**Pull request by srinathkr07**
Thanks for contributing your submission to the e-Rum2020 CovidR contest.

**For CovidR contributors:**

- [x] The title of this PR reflects your contribution
- [x] The pull request only includes one new `.yml` file under `contest/contributions`
- [x] Make sure "Allow edits from maintainers" is checked
- [x] Send an email to covidrcontest@erum.io (for further direct communication down the road)
     - Subject: "Submission to CovidR: PR #00", where #00 is the number of this Pull Request
     - Content: A full link to this Pull Request
     - Optionally include your social media profiles (Twitter, LinkedIn, Facebook): we will use them to `@mention` you while sharing the contribution through our channels.

---

For CovidR maintainers:

- [x] Label the PR as "submission"
- [x] Check the validity of the submission
- [x] Assess completeness of metadata and possibly adjust the `.yml` file-name
- [x] Check the integration of the new page in the website gallery (via Actions' artifact or locally)
- [x] Make sure the email to covidrcontest@erum.io was received
- [x] Add a field with the current `date` and approx. merge time (e.g. `2020-04-01-16:30` for April 1st, 4:30 PM)
- [x] Squash-merge the pull request (commit headline will be OK, consider cleaning-up the details)
- [x] The initial comment should be automatically created, otherwise comment manually with the raw content of [vote-comment.md](../tree/master/contest/_tools/vote-comment.md) and an initial like
- [x] Provide here the link to the contribution as published to the gallery: https://milano-r.github.io/erum2020-covidr-contest/srinathkr-covid19-in-india.html
- [x] Instruct contributors on how to include their badge from the gallery page, share on social media, and get a [free ticket](https://www.eventbrite.it/e/covidr-erum-2020-pre-conference-event-tickets-104133026686).
- [x] Consolidate e-mailed information.

### CODE REVIEW

**Review by riccardoporreca — CHANGES_REQUESTED** [review]
@srinathkr07, thanks for you submission, can you please make sure the abstract is reduced to the requested 800 characters (including spaces)? It currently overshoots by some 350 characters.

**Comment by riccardoporreca** [comment]
@srinathkr07, on a side note, when assessing the integration of your contribution into the CovidR Gallery, I noticed the map can be cut for "wide" window geometries, e.g. 
image

You don't have to fix this before we accept your submission, but you may want to consider how to address this in the coming days, perhaps by simply allowing to zoom out (and drag) the map via`opts_zoom(min = 0, max = 1)` in
https://github.com/srinathkr07/COVID-19-in-India/blob/e1413b59ad4c572ffd4a06a718cf7ef8958ab82f/India.Rmd#L233

**Comment by srinathkr07** [comment]
> @srinathkr07, thanks for you submission, can you please make sure the abstract is reduced to the requested 800 characters (including spaces)? It currently overshoots by some 350 characters.

Done! Can you check the latest abstract?

**Comment by riccardoporreca** [comment]
Thanks @srinathkr07 for the prompt action, I am going to finalize the submission process.

File: contest/contributions/srinathkr-covid19-in-india.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  The data source is COVID-19 India API, a volunteer-driven API for COVID-19 stats in India which collects data from official press bulletins and reports released by various states.
+  This project provides a summary of all types of cases reported, for the entire country in total and for each state; provides state-wise analysis with the help of a plot and a map and provides both cumulative and daily trends in cases. 
+  The main feature of this project is the SIR (Susceptible - Infectious - Recovered) model. This model has been applied for India for a period of 150 days starting from March 1, 2020. Based on this, R0 is also calculated.
+  
+menu_entry: COVID-19 in India
+repository: https://github.com/srinathkr07/COVID-19-in-India/
+type: shiny
+content: https://srinathkr.shinyapps.io/COVID19-India/
+iframe_resizer: false
+date: 2020-05-19-15:00
```
Comment (riccardoporreca) [inline contest/contributions/srinathkr-covid19-in-india.yml]: Finalize acceptance `date`
```suggestion
date: 2020-05-19-17:30
```

**Comment by srinathkr07** [comment]
Anything else to be changed?
Also, what does "Do not us responsive iframe" mean?

**Review by riccardoporreca — APPROVED** [review]

**Comment by riccardoporreca** [comment]
>  Also, what does "Do not us responsive iframe" mean?

Don't worry about this, it is a technical detail of the way we embed pages, which for certain cases we have to disable for  a better look.

**Comment by riccardoporreca** [comment]
@srinathkr07, your submission is now accepted and visible at https://milano-r.github.io/erum2020-covidr-contest/srinathkr-covid19-in-india.html :tada: 
You can click on the **eRum2020::Covidr badge** in the page to obtain the badge code you can embed in your repository.

Furthermore, feel free to spread your contribution on social media with the link above, using hashtags `#erum2020` `#covidr`, and invite the community to like the contribution using the corresponding button.

Last but not least: You can get your **free ticket for the pre-conference event** at https://www.eventbrite.it/e/covidr-erum-2020-pre-conference-event-tickets-104133026686

**Pull request merged**